governance.md

```diff
@@ -59,260 +70,170 @@ The foundations of Project governance are:
-membership again at any time in the future, like any other Project Contributor.
-Retired Core Team members will be listed on the project website, acknowledging
-the period during which they were active in the Core Team.
-
-The Core Team reserves the right to eject current Members, other than the BDFL,
-if they are deemed to be actively harmful to the project’s well-being, and
-attempts at communication and conflict resolution have failed.
+To manage the project, there are different Teams that each have responsibility for
+specific aspects of the project. Collectively, the members of all Teams are referred to
+as _Stewards_ of the project.
+
+## Steering Committee
+
+The role of the _pandas_ Steering Committee is to coordinate the activities of the
+different Teams and to ensure that different policies and procedures are carried out in
+a consistent manner. The Steering Committee has no routine decision-making authority,
+except as detailed herein, although in exceptional circumstances it may be called upon
+from time to time to make decisions that are in the best interest of The Project as a
+whole. The Steering Committee will itself decide when a circumstance is exceptional.
+
+The Steering Committee may create a working group to consider changes to the governance
+model described in this document, including the creation of new Teams that support the
+goals of The Project. Changes to the governance model will require a voting process and
+approval equivalent to the process described in PDEP-1, with the exception that all
+Stewards are eligible to vote.  
+
+The Steering Committee may appoint temporary working groups to work on issues, such as
+governance, that fall outside the scope of responsibilities for the existing Teams.
+
+The ideal composition of the Steering Committee consists of 5 people.  The Steering
+Committee will be chosen via an approval voting process from a slate of nominees,
+meaning that each person who is eligible to vote may vote for more than one candidate
+from the slate.  
+
+### Initial Election
+
+The initial slate for the Steering Committee election will consist of 5 or more
+candidates who volunteer to be on the slate of candidates for the initial Steering
+Committee.  For the initial election, any candidate receiving more than 50% of the total
+number of members of the previous _pandas_ core team that register at least one vote
+will then become a member of the initial Steering Committee. 
+
+### Subsequent Elections  
+
+Subsequent changes to the membership of the Steering Committee will occur towards the
+end of each calendar year.  By November 30 of each year, each member of the Steering
+Committee will be asked if they would like to continue in that role. If any member
+decides to step down from the Steering Committee, new volunteers will be solicited from
+the group of stewards by the Steering Committee.  Those new volunteers will then be on a
```

Comment:
The idea here was that members of the Steering Committee, once elected, can remain on the Steering Committee for as long as they want, and they don't have to get re-elected each year.  I think (but I'm not sure) this is how other projects do this.

Reply:
We can consider putting up the entire Steering Committee up for re-election each year, if they want to continue.

Reply:
>  once elected, can remain on the Steering Committee for as long as they want,

My main comment here that I personally think we should not allow the situation that someone can decide themselves to essentially stay on the committee indefinitely, without ever being reelected. That doesn't seem like a healthy situation if that would ever occur. At least putting a year limit on it would prevent that, but also just requiring to be reelected every election is also a possibility.

Reply:
clarified the language here that the Steering Committee members who do not wish to step down are also on the slate for the election.